ext/standard/tests/time/001.phpt

```diff
@@ -1,9 +1,8 @@
 --TEST--
 microtime() function
 --SKIPIF--
-<?php 
-	if (!function_exists('microtime'))  die('skip microtime() not available'); 
-	die('warn system dependent');
```

Comment:
Remove this `die` as IMO have no value to this test

Reply:
hmm did this test always `die` and therefore never run the actual test logic?

Reply:
@staabm Nops, to skip a test with `die`, `print`, `echo`, etc, it must starts with `skip` :smile:

Reply:
oh wow, we create a php process for each `SKIPIF` we find in the testsuite... sounds like some severe process overhead.. maybe a good area to explore a alternative to safe resources on CI builds/running testsuite etc.